### ISSUE TITLE
agent off on instances, instances are not replaced

### ISSUE THREAD

**Issue by frsechet** [convox/rack#1098]
Trying to deploy an (already existing) app to convox, I had the following deployment log:
```
... docker build is all fine until the very end and then...
build │ 4232687742d5: Pushed
build │ cf27ae36fa00: Pushed
build │ 2f71b45e4e25: Pushed
build │ main.BQULMXNMEIU: digest: sha256:02650af04bbae3b1b9bf677770934ce28728b3b88d85aec3e6e00fcf4916ff69 size: 9006
2016/08/19 09:44:46 wait: no child processes
ERROR: my-app build failed
```

Cloudformation doesn't show any error, everything looks successfull.
I also receive a "created release XXXXXXX" notification, but the app doesn't deploy.

`convox rack ps` more or less timeouts:
```
ERROR: Get https://console.convox.com/apps/my-rack/processes?stats=false: EOF
```

`convox instances` shows instance agents as off:
```
ID          AGENT  STATUS  STARTED       PS  CPU     MEM
i-c59e434e  on     active  9 hours ago   4   4.76%
i-90625e1c  off    active  21 hours ago  5   99.64%
i-3420deb9  off    active  23 hours ago  7   39.04%
```

After *manually* killing both apparently failing instances, everything looks right again, and my app correctly deploys.

But:

- why 99% CPU? Nothing really heavy running on those instances, it rather looks like a stuck process
- why aren't the instances automatically replaced?

**Comment by ddollar** [comment]
When we looked his monitor process was dead. Debugging.

**Comment by frsechet** [comment]
monitor process was dead because it is set to min=0%, max=100%, so not an "error" for ECS. Should probably be 50%-150% if it is an essential part of convox?

**Comment by ddollar** [comment]
Fixed by #1105

**Comment by MiguelMoll** [comment]
The monitor's deployment percentage was release with [20160819234527](https://github.com/convox/rack/releases/tag/20160819234527)

We'll track the build error with #1100. Thanks for the detailed report. 👍
